InfrastructureManager/agents/factory.py

```diff
@@ -1,7 +1,9 @@
+from agents.azure_agent import AzureAgent
```

Comment:
No sadly the agent code on the AppScale side is different from the Tools. Eventually I will have to look at what we need to just do the import but all the agents are used a little differently on the AppScale side.